pkg/agent/openflow/network_policy.go

```diff
@@ -1823,13 +1827,63 @@ func newFeatureNetworkPolicy(
 }
 
 func (f *featureNetworkPolicy) initFlows() []binding.Flow {
-	f.egressTables = map[uint8]struct{}{EgressRuleTable.GetID(): {}, EgressDefaultTable.GetID(): {}}
+	f.egressTables = map[uint8]struct{}{EgressDefaultTable.GetID(): {}}
 	if f.enableAntreaPolicy {
 		f.egressTables[AntreaPolicyEgressRuleTable.GetID()] = struct{}{}
 	}
 	var flows []binding.Flow
-	flows = append(flows, f.establishedConnectionFlows()...)
-	flows = append(flows, f.relatedConnectionFlows()...)
-	flows = append(flows, f.ingressClassifierFlows()...)
+	if f.nodeType == config.K8sNode {
+		f.egressTables[EgressRuleTable.GetID()] = struct{}{}
+		flows = append(flows, f.ingressClassifierFlows()...)
+	}
+	flows = append(flows, f.skipPolicyRuleCheckFlows()...)
+	return flows
+}
+
+// skipPolicyRuleCheckFlows installs OpenFlow entries to resubmit the packets in an established or related
+// connections to the metric table in the same stage directly. So that these packets would skip the flows
+// to check NetworkPolicy rules.
```

Comment:
```suggestion
// skipPolicyRuleCheckFlows generates the flows to forward the packets in an established or related
// connections to the metric table in the same stage directly, so that these packets can skip the flows
// for NetworkPolicy rules.
```